multinode/node_lifecycle.go

```diff
@@ -567,7 +567,7 @@ func (n *node[CHAIN_ID, HEAD, RPC]) unreachableLoop() {
 
 			err := n.rpc.Dial(ctx)
 			if err != nil {
-				lggr.Errorw(fmt.Sprintf("Failed to redial RPC node; still unreachable: %v", err), "err", err, "nodeState", n.getCachedState())
+				lggr.Warnw(fmt.Sprintf("Failed to redial RPC node; still unreachable: %v", err), "err", err, "nodeState", n.getCachedState())
```

Comment:
Why is it necessary to change the log level?

Reply:
Found the answer. I'm not sure that we should do it even if external RPC's might be unreliable. NOPs on different setups might use alters based on the errors. 
It would be nice to look deeper into issues that NOPs observed with third-party RPCs. We might be able to provide them with a config to reduce number false alerts